src/editor/blocks/posts-inserter/utils.js

```diff
@@ -61,6 +61,14 @@ const getExcerptBlockTemplate = ( post, { excerptLength, textFontSize, textColor
 	return [ 'core/paragraph', assignFontSize( textFontSize, attributes ) ];
 };
 
+const getContinueReadingLinkBlockTemplate = ( post, { textFontSize, textColor } ) => {
+	const attributes = {
+		content: `<a href="${ post.link }">${ __( 'Continue reading…', 'newspack' ) }</a>`,
```

Comment:
Since the Posts Inserter block can be turned into regular blocks, I think this covers customisation to an extent. Customisation via block attribute can be added later if need be.

Reply:
Good point, but I bet it will be a request soon enough. I'm okay with leaving it for now.